sci-mathematics/mathematica/mathematica-13.0.1.ebuild

```diff
@@ -0,0 +1,140 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+CHECKREQS_DISK_BUILD=19G
+inherit desktop unpacker xdg
+
+DESCRIPTION="Wolfram Mathematica"
+SRC_URI="Mathematica_${PV}_BNDL_LINUX.sh"
+HOMEPAGE="https://www.wolfram.com/mathematica/"
+
+LICENSE="all-rights-reserved"
+KEYWORDS="-* ~amd64"
+SLOT="0"
+IUSE="cuda +doc"
+
+RESTRICT="strip mirror bindist fetch"
+
+# Mathematica comes with a lot of bundled stuff. We should place here only what we
+# explicitly override with LD_PRELOAD.
+# RLink (libjri.so) requires dev-lang/R
+RDEPEND="
+	dev-lang/R
+	cuda? ( dev-util/nvidia-cuda-toolkit )
+	media-libs/freetype
+	virtual/libcrypt
+"
+
+DEPEND="
+	${RDEPEND}
+"
+
+BDEPEND="
+	dev-util/patchelf
+"
+
+# we need this a few times
+MPN="Mathematica"
+MPV=$(ver_cut 1-2)
+M_BINARIES="MathKernel Mathematica MathematicaScript WolframKernel WolframScript math mathematica mcc wolfram"
+M_TARGET="opt/Wolfram/${MPN}/${MPV}"
+
+# we might as well list all files in all QA variables...
+QA_PREBUILT="opt/*"
+
+S=${WORKDIR}
+
+src_unpack() {
+	/bin/sh "${DISTDIR}/${A}" --nox11 --keep --target "${S}/unpack" -- "-help" || die
+}
+
+src_prepare() {
+	pushd "${S}/unpack" > /dev/null || die
+	# fix ACCESS DENIED issue when installer check the avahi-daemon
+	sed -e "s|avahi-daemon -c|true|g" -i "Unix/Installer/MathInstaller" || die
+	/bin/sh "Unix/Installer/MathInstaller" -auto "-targetdir=${S}/${M_TARGET}" "-execdir=${S}/opt/bin" || die
+	popd > /dev/null || die
+	eapply_user
+}
+
+src_install() {
+	local ARCH='-x86-64'
+
+	if ! use doc; then
+		einfo "Removing documentation"
+		rm -r "${S}/${M_TARGET}/Documentation" || die
+	fi
+
+	einfo 'Removing MacOS- and Windows-specific files'
+	find "${S}/${M_TARGET}" -type d -\( -name Windows -o -name Windows-x86-64 \
+		-o -name MacOSX -o -name MacOSX-x86-64 -o -name Macintosh -\) \
+		-exec rm -rv {} + || die
+
+	if ! use cuda; then
+		einfo 'Removing cuda support'
+		rm -r "${S}/${M_TARGET}/SystemFiles/Components/CUDACompileTools/LibraryResources/Linux-x86-64/CUDAExtensions.so" || die
+	fi
+
+	# Linux-x86-64/AllVersions is the supported version, other versions remove
+	einfo 'Removing unsupported RLink versions'
+	rm -rf "${S}/${M_TARGET}/SystemFiles/Links/RLink/SystemFiles/Libraries/Linux-x86-64/3.5.0" || die
```

Comment:
We shouldn't use `-f` on rm, because it makes `|| die` not do anything since the exit status will always be success.

Reply:
Removed -f